core/.js/src/test/scala/io/udash/bindings/inputs/TextAreaTest.scala

```diff
@@ -222,5 +257,21 @@ class TextAreaTest extends AsyncUdashFrontendTest {
         }
       }
     }
+
+    "not run callback when value is the same with debouncing" in {
+      val p = Property[String]("ABC")
+      var called = false
+      val input = TextArea(p, 1 seconds, _ => called = true)()
+      val inputEl = input.render
+
+      called should be(false)
+
+      inputEl.changeValue("CBA")
+      inputEl.changeValue("ABC")
+
+      waiting({
```

Comment:
I think you can test this more thoroughly without adding a new utility:
- improve the callback to append text to a list
- just use retrying with should not be empty and a time limit - you can map over the result and treat failure as a success

should be a matter of the right `PatienceConfig`

Reply:
Isn't it the same as current solution but more confusing on the first glance? What would be the gain from that?